node/src/GlideClusterClient.ts

```diff
@@ -314,6 +318,151 @@ export class GlideClusterClient extends BaseClient {
         );
     }
 
+    /**
+     * @internal
+     */
+    protected scanOptionsToProto(
+        cursor: string,
+        options?: ScanOptions,
+    ): command_request.ClusterScan {
+        const command = command_request.ClusterScan.create();
+        command.cursor = cursor;
+
+        if (options?.match) {
+            command.matchPattern = Buffer.from(options.match);
+        }
+
+        if (options?.count) {
+            command.count = options.count;
+        }
+
+        if (options?.type) {
+            command.objectType = options.type;
+        }
+
+        return command;
+    }
+
+    /**
+     * @internal
+     */
+    protected createClusterScanPromise(
```

Comment:
and this to base client

Reply:
Same. Base is the parent, it should include what relevant for both, this is a method belong to cluster, it shouldn't be available on base or standalone scope.